scripts/globals/status.lua

```diff
@@ -1906,7 +1906,7 @@ xi.latent =
     -- 41 free to use
     -- 42 free to use
     WEAPON_DRAWN_HP_UNDER    = 43, -- PARAM: HP PERCENT
-    -- 44 free to use
+    HP_BASEU_TP_UNDER_100    = 44, -- Base HP (no convert or +% taken into account) <= %, TP < 100
```

Comment:
Baseu?

Reply:
`BASEU[NDER]`. Not married to this name at all but wanted to keep the max latent effect name length the same, admittedly not a fantastic reason.

Reply:
I'd say make it more readable personally, but not going to block on it.

Reply:
Yeah I agree with you, I'll get to it after the weekend!